### PR TITLE
Pad page

### PR DESCRIPTION
**Pull request by arisslin**
- Show padname on padPage
- Create VolumeSlider component
- Adjust master volume with VolumeSlider on main page
- storybook
- add proptypes to components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ export default function App() {
     setPadIsTriggered(key, true)
     const padName = getPadNameByKey(key)
     setSelectedPad(padName)
+    //.. setSelectedSample(samplePlayer -> key)
```
Comment (evabrinkmann) [inline src/App.js]: Is this comment still needed? Maybe you forgot to delete it.

File: src/common/SamplePlayer.js [hunk 2]
```diff
@@ -14,6 +14,10 @@ export default class SamplePlayer {
     return this._pads
   }
 
+  getMasterVolume() {
+    return this.samplePlayers.volume.value
+  }
+
```
Comment (evabrinkmann) [inline src/common/SamplePlayer.js]: How come you work with class based components here? Just curious to know.

File: src/components/DisplayPages/DisplayPageMain.js [hunk 3]
```diff
@@ -3,11 +3,21 @@ import DisplayPageTitle from './DisplayPageTitle'
 import styled from 'styled-components/macro'
 
 export default function DisplayPageMain({ masterVolume }) {
+  console.log(masterVolume)
```
Comment (evabrinkmann) [inline src/components/DisplayPages/DisplayPageMain.js]: Maybe you forgot to delete the console.log  here?

File: src/components/DisplayPages/DisplayPageMain.js [hunk 4]
```diff
@@ -16,7 +20,46 @@ export default function DisplayPageMain({ masterVolume, onChangeInputVolume }) {
           defaultValue={masterVolume}
           onChange={onChangeInputVolume}
         />
-      </label>
+      </label> */}
```
Comment (evabrinkmann) [inline src/components/DisplayPages/DisplayPageMain.js]: Whats happening btw. line 13 and 23 ? Kind of commented out ... Maybe something to delete?

File: src/components/DisplayPages/DisplayPageMain.js [hunk 5]
```diff
@@ -25,11 +68,26 @@ const DisplayPageMainStyled = styled.article`
...
 
-  label[for='master-volume'] {
+  /* label[for='master-volume'] {
     span {
       display: inline-block;
       width: 58px;
       text-align: right;
     }
-  }
+  } */
```
Comment (evabrinkmann) [inline src/components/DisplayPages/DisplayPageMain.js]: I'm confused. /*  */usually tells that this part is commented out but it isn't, is it?

File: src/components/Display.js [hunk 6]
```diff
@@ -5,6 +5,14 @@ import DisplayPageInfo from './DisplayPages/DisplayPageInfo'
 import DisplayPageMain from './DisplayPages/DisplayPageMain'
 import DisplayPagePad from './DisplayPages/DisplayPagePad'
 
+import PropTypes from 'prop-types'
+
+Display.propTypes = {
+  selectedPad: PropTypes.string,
+  masterVolume: PropTypes.number.isRequired,
```
Comment (evabrinkmann) [inline src/components/Display.js]: I get a warning in the console: Failed prop type: Invalid prop `masterVolume` of type `string` supplied to `Display`, expected `number`. 
That's weird as you declared it as a number.

File: src/components/DisplayPages/DisplayPageMain.js [hunk 7]
```diff
@@ -2,6 +2,12 @@ import React from 'react'
 import DisplayPageTitle from './DisplayPageTitle'
 import styled from 'styled-components/macro'
 import VolumeSlider from '../../components/VolumeSlider'
+import PropTypes from 'prop-types'
+
+DisplayPageMain.propTypes = {
+  masterVolume: PropTypes.number.isRequired,
```
Comment (evabrinkmann) [inline src/components/DisplayPages/DisplayPageMain.js]: Here I get the same warning as well: Failed prop type: Invalid prop `masterVolume` of type `string` supplied to `DisplayPageMain`, expected `number`.

File: src/components/VolumeSlider.js [hunk 8]
```diff
@@ -1,5 +1,12 @@
 import React from 'react'
 import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+VolumeSlider.propTypes = {
+  label: PropTypes.string.isRequired,
+  masterVolume: PropTypes.number.isRequired,
```
Comment (evabrinkmann) [inline src/components/VolumeSlider.js]: Here as well: Failed prop type: Invalid prop `masterVolume` of type `string` supplied to `VolumeSlider`, expected `number`.

File: src/common/SamplePlayer.js (hunk 2 above)
Comment (arisslin) [reply src/common/SamplePlayer.js]: Oh, it is not a class based componend, SamplePlayer is a class itself outside of react. ;-)

File: src/App.js (hunk 1 above)
Comment (arisslin) [reply src/App.js]: Yes, thanks i forgot it.

**Pull request merged**